CMakeLists.txt

```diff
@@ -51,9 +51,8 @@ find_package(re2 REQUIRED)
 # Includes
 # ---------------------------------------------------------------------------
 
-include_directories(SYSTEM
+include_directories(
         ${CMAKE_SOURCE_DIR}/include
-        ${roaring_INCLUDE_DIRS}
```

Comment:
Why is this change?

Reply:
`${roaring_INCLUDE_DIRS}`

was unneeded because the `find_package` scripts and cmake did this already.

`SYSTEM` hid warnings from the specified include directories, as the flag is supposed to be used for system libraries which you cannot edit